mono/mini/mini-exceptions.c

```diff
@@ -725,6 +725,8 @@ unwinder_unwind_frame (Unwinder *unwinder,
 		if (frame->type == FRAME_TYPE_INTERP_TO_MANAGED) {
 			unwinder->in_interp = TRUE;
 			mini_get_interp_callbacks ()->frame_iter_init (&unwinder->interp_iter, frame->interp_exit_data);
+			parent = mini_get_interp_callbacks ()->frame_get_parent (frame->interp_exit_data);
+			MONO_CONTEXT_SET_SP (new_ctx, parent);
```

Comment:
does this fix the issue regarding mixed mode?

Reply:
this fixes thead6.exe. The issue with mixed mode is more complex